R/ui_panels/example_tab_1.R

```diff
@@ -8,48 +8,42 @@ example_tab_1_panel <- function() {
           id = "main_col",
           h1("Overall content title for this dashboard page"),
         ),
-        # Expandable section --------------------------------------------------
+        # Non-Expandable section --------------------------------------------------
```

Comment:
Maybe call this something like 
```suggestion
        # Inputs section --------------------------------------------------
```
instead